includes/admin/settings/register-settings.php

```diff
@@ -477,7 +477,17 @@ function mdjm_get_registered_settings()	{
 										'to help build an information library. The consolidated list of playlist songs will be freely shared. ' . 
 										'Only song, artist and the %s type information is transmitted.', 'mobile-dj-manager' ), mdjm_get_label_singular( true ) ),
 						'type'        => 'checkbox'
+					),
+					'playlist_limit' => array(
+						'id'          => 'playlist_limit',
+						'name'        => __( 'Playlist Limit?', 'mobile-dj-manager' ),
+						'desc'        => sprintf( __( 'If this number is non-zero, event playlists will be limited to this number of entries ', 'mobile-dj-manager' ), mdjm_get_label_singular( true ) ),
```

Comment:
We shouldn't ever output the words 'event' or 'events' as this would mean that we would break functionality for people who are hooking into the `mdjm_default_events_name` which allows the word 'event' to be replaced with whatever term a business prefers. Instead we use `mdjm_get_label_singular()` and `mdjm_get_label_plural()` for 'event' and 'events' respectively. Default is that they will have a capital 'E'. To fix to lowercase, pass 'true' to either function.

With that in mind, your `sprintf` command should be something like...

`sprintf( __( 'If this number is non-zero, %s playlists will be limited to this number of entries ', 'mobile-dj-manager' ), mdjm_get_label_singular( true ) ),`